mkt/api/authorization.py

```diff
@@ -177,6 +165,40 @@ def __call__(self, *a):
         return self
 
 
+class GroupPermission(GroupPermissionBase):
+
+    def __init__(self, app, action):
+        self.app = app
+        self.action = action
+
+
+class AllowReviewerReadOnly(GroupPermissionBase):
+    app = 'Apps'
+    action = 'Review'
+
+    def has_permission(self, request, view):
+        return (request.method in SAFE_METHODS and
+                super(AllowReviewerReadOnly, self).has_permission(request,
+                                                                  view))
+
+
+class AllowStaffReadOnlyIfReviewed(GroupPermissionBase):
+    """
+    Allow users in the Staff group to view apps that have any published status.
+    """
+    app = 'Apps'
+    action = '%'
```

Comment:
From the name of the class I thought the intention was to have a more restrictive than AllowReviewerReadOnly group, where it was only Staff.  If it was supposed to be really inclusive then its fine (I didn't look to see where it was used)